content/fr/developers/write_agent_check.md

```diff
@@ -42,7 +42,7 @@ try:
     # Premier essai d'importation de la classe de base à partir des anciennes versions de l'Agent…
     from checks import AgentCheck
 except ImportError:
-    # …si la commande ci-dessus a échoué, le check s'exécute dans l'Agent version 6 ou ultérieure
+    # …si la commande ci-dessus a échoué, le check s'exécute dans l'Agent version 6 ou ultérieure
```

Comment:
For some reason git considers the space has changed, but I can't put it back to what is was (`git diff` does not show anything)

Reply:
This should be ok. Let us know when you are ready to merge.

Reply:
@ruthnaebeck this PR can be merged